exonum-java-binding/core/rust/exonum-java/build.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::env;
+
+fn main() {
+    let lib_paths = [
+        "ROCKSDB_LIB_DIR",
+        "SNAPPY_LIB_DIR",
+        "LZ4_LIB_DIR",
+        "BZ2_LIB_DIR",
+        "Z_LIB_DIR",
+    ];
+
+    lib_paths
+        .iter()
+        .filter_map(|lib| env::var(lib).ok())
+        .for_each(|path| println!("cargo:rustc-search-lib=native={}", path));
```

Comment:
I am not sure I understand why this is needed. I'd clarify the intent of this code (why does it allow to specify the library search paths if these env variables are not used in the project build configuration?).